.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
```

Comment:
We don't need this, `ci.yaml` already does this for us

Reply:
Removed.